core/app/api/v0/c/[communitySlug]/site/[...ts-rest]/route.ts

```diff
@@ -350,7 +350,7 @@ const handler = createNextHandler(
 				const { user, community } = await checkAuthorization({
 					token: { scope: ApiAccessScope.pub, type: ApiAccessType.read },
 					// TODO: figure out capability here
-					cookies: false,
+					cookies: true,
```

Comment:
I'm not sure what's supposed to go here but I imagine it shouldn't just be `true` 😅

Reply:
hmm true might actually be fine i think! we filter the pubs by the user anyway, so i don't think allowing blanket access to this endpoint is that bad. i think maybe we should at least check that the user is a member of the community (i think currently `cookies: true` just means that the user must be logged in). as extra work, could you maybe add an option for cookies to be set to "community-member", and if it is, it will check in `checkAuthorization` if the user object returned from `getLoginData` has a membership for the current community on it?

Reply:
(dont do this for this pr, you can do that in the follow up if you want)